pkg/gofr/datasource/cassandra/cassandra.go

```diff
@@ -177,6 +176,40 @@ func (c *Client) ExecCAS(dest interface{}, stmt string, values ...interface{}) (
 	return applied, err
 }
 
+func (c *Client) NewBatch(batchType int) error {
+	const (
+		loggedBatch = iota
+		unloggedBatch
+		counterBatch
+	)
+
+	var err error
+
+	switch batchType {
+	case loggedBatch, unloggedBatch, counterBatch:
+		c.cassandra.batch = c.cassandra.session.newBatch(gocql.BatchType(batchType))
+
+	default:
+		err = errUnsupportedBatchType
+	}
+
+	return err
+}
+
+func (c *Client) BatchQuery(stmt string, values ...interface{}) {
+	c.cassandra.batch.Query(stmt, values...)
+}
+
+func (c *Client) ExecuteBatch() error {
+	defer c.postProcess(&QueryLog{Query: "", Keyspace: c.config.Keyspace}, time.Now())
```

Comment:
Is there a need to define query to its default value?

```suggestion
	defer c.postProcess(&QueryLog{Keyspace: c.config.Keyspace}, time.Now())
```

Except if you enforcing via exhaust struct linter or equivalent, I don't see why.